packages/loopring_v3/contracts/impl/owners/LoopringIOExchangeOwner.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2017 Loopring Technology Limited.
+pragma solidity ^0.6.10;
+pragma experimental ABIEncoderV2;
+
+import "../../iface/IExchangeV3.sol";
+
+import "../../lib/LzDecompressor.sol";
+
+import "../../thirdparty/BytesUtil.sol";
+
+import "./SelectorBasedAccessManager.sol";
+
+contract LoopringIOExchangeOwner is SelectorBasedAccessManager
+{
+    using BytesUtil for bytes;
+
+    bool   public open;
+
+    event SubmitBlocksAccessOpened(bool open);
+
+    constructor(address _exchange)
+        public
+        SelectorBasedAccessManager(_exchange)
+    {
+    }
+
+    function submitBlocksCompressed(
+        bytes calldata data
+        )
+        external
+    {
+        bytes4 selector = IExchangeV3(0).submitBlocks.selector;
```

Comment:
@Brechtpd Ideally I want to use a constant selector instead of doing this read. How to calculate/find this selector?

Reply:
This is a sample: bytes4(keccak256("isValidSignature(bytes,bytes)")

Reply:
How about functions with Structure parameters.

Reply:
> Ideally I want to use a constant selector instead of doing this read

It's not a read, the value is calculated at compile time. There's no difference in cost so I don't see any reason not to use it.